src/registry.test.ts

```diff
@@ -0,0 +1,17 @@
+import { SkynetClient, defaultSkynetPortalUrl, genKeyPairAndSeed } from "./index";
+import { hashDataKey } from "./crypto";
+
+const portalUrl = defaultSkynetPortalUrl;
+const client = new SkynetClient(portalUrl);
+const { publicKey } = genKeyPairAndSeed();
+const dataKey = "app";
+
+describe("getEntryUrl", () => {
+  it("should generate the correct registry url for the given entry", () => {
+    const url = client.registry.getEntryUrl(publicKey, dataKey);
+    const encodedPK = encodeURIComponent(`ed25519:${publicKey}`);
+    const encodedDK = encodeURIComponent(Buffer.from(hashDataKey(dataKey)).toString("hex"));
```

Comment:
I would actually hardcode those as strings because if something changes in Buffer in next node or something, we will not catch an issue.